### ISSUE TITLE
Investigate inkcyclopedia spreadsheet automatic update

### ISSUE THREAD

**Issue by coffeepenbit** [terminalnode/mrfreeze#6]
Investigate if the inkcyclopedia on airtable can be automatically implemented

**Comment by terminalnode** [comment]
Progress on this currently blocked by: https://github.com/terminalnode/mrfreeze/issues/12

**Comment by terminalnode** [comment]
No longer relevant with the new api search.
